### ISSUE TITLE
[BUG] CRC VM does not start on Windows 10 - ERRO Error running post start: Nameserver not successfully set

### ISSUE THREAD

**Issue by ralf-hoch** [code-ready/crc#1725]
### General information

  * OS: Windows 10
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)? YES
  * Running CRC on: Laptop

## CRC version
```bash
# Put `crc version` output here
```
CodeReady Containers version: 1.17.0+99f5c87
OpenShift version: 4.5.14 (embedded in binary)
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
```
DEBU CodeReady Containers version: 1.17.0+99f5c87
DEBU OpenShift version: 4.5.14 (embedded in binary)
DEBU About to run SSH command with hidden output
DEBU SSH command failed
DEBU
DEBU About to run SSH command:
df -B1 --output=size,used,target /sysroot | tail -1
DEBU SSH command results: err: <nil>, output: 32720793600 10620887040 /sysroot
CRC VM:          Running
OpenShift:       Running (v4.5.14)
Disk Usage:      10.62GB of 32.72GB (Inside the CRC VM)
Cache Usage:     16.18GB
Cache Directory: C:\Users\ralfh\.crc\cache
## CRC config
```bash
# Put `crc config view` output here
```
command does not provide any output
## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
```
https://gist.github.com/ralf-hoch/ec5c78f20f0f70c267fef25367d28948
### Steps to reproduce

  1. crc setup
  2. crc start --log-level debug

### Expected
crc start exits with success

### Actual
crc exits with this error: ERRO Error running post start: Nameserver not successfully set

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

The command sequence suggested above has been used but does not change the behaviour.

https://gist.github.com/ralf-hoch/fb517e33ef359aabc340df3d49fbd592

crc start did already work before. It looks like a recent MS patch maybe introduced the issue, but I was not able to identify the related patch yet.

**Comment by guillaumerose** [comment]
Last crc version introduced a new network mode that should fix your issue. Can you give it a try ?
https://github.com/code-ready/crc/wiki/VPN-support--with-an--userland-network-stack
Thanks.

**Comment by ralf-hoch** [comment]
Hi Guillaume - I will try but it will take some time. For unknown reason downloading from the mirror is awfully slow for me at the moment. Estimate is about 22 hrs to download the 2.6 GB. I will test it as soon as the download completed.

**Comment by gbraad** [comment]
Duplicate of #1193 
Workaround are given in that issue

**Comment by gbraad** [comment]
> For unknown reason downloading from the mirror is awfully slow for me at the moment. For unknown reason downloading from the mirror is awfully slow for me at the moment. 

Restarting the download might help, as different locations are available. You might have to remove a session (as sticky session is used).
